README.md

```diff
@@ -1,24 +1,52 @@
-# README
+# Recipe
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app should allow you to make them public so anyone can access them.
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+-  Ruby on Rails
+-  PostgreSQL
 
-* System dependencies
+## Live Demo
+[Live Demo](https://mustbudgetapp.herokuapp.com/)
 
-* Configuration
+## Video 
+[Loom video](https://www.loom.com/share/aa32656c0c094bff814ee49ba0ef2adf)
 
-* Database creation
+## Screen 
+![screenshot](./ScreenShot1.png)
+![screenshot](./ScreenShot2.png)
 
-* Database initialization
+## Getting Started
 
-* How to run the test suite
+Download the ZIP from this [Link](https://github.com/mustabbas/budget-app.git), or run the following git command to clone the files to your machine:
 
-* Services (job queues, cache servers, search engines, etc.)
+- git clone https://github.com/mustabbas/budget-app.git
+- Open the cloned folder on the IDE of your choice
+- Install the required dependecies with `bundle install`
+- run `db:migrate`
+- Use `rails s `to run the program
+
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.